engine/src/flutter/shell/platform/windows/task_runner_window.cc

```diff
@@ -115,6 +130,11 @@ TaskRunnerWindow::HandleMessage(UINT const message,
                                 LPARAM const lparam) noexcept {
   switch (message) {
     case WM_TIMER:
+      if (wparam == kPollTimeoutTimerId) {
+        // Ignore PollOnce timeout timer.
+        return 0;
+      }
+      FML_DCHECK(wparam == kTimerId);
```

Comment:
The fall-through behavior isn't immediately obvious, could we duplicate the `ProcessTasks();` + `return 0;` logic here?

Reply:
Good point. Every time I see fall-through like that I'm wondering a bit if it was by accident...